ai-ml/llm-serving-gemma/vllm/vllm-2b-awq/patch.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: vllm-gemma-deployment
+spec:
+  template:
+    metadata:
+      labels:
+        ai.gke.io/model: gemma-2b-awq
+    spec:
+      containers:
+      - name: inference-server
+        args:
+        - --model=$(MODEL_ID)
+        - --tensor-parallel-size=1
+        - --quantization=awq
+        env:
+        - name: MODEL_ID
+          value: dganochenko/gemma-2b-AWQ
```

Comment:
Please change this to `value: google/gemma-2b-AWQ` so it points to the right repository.

Reply:
it's done already